e2e/sponsor.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+import { generateUUID } from "./utilties"
+
+
+test('Fill in sponsor form twice', async ({ page }) => {
```

Comment:
Normally, I'd have a test for `filling in the sponsor form once` and then another one for `filling it in twice` to break it down into 2 separate use cases, but I think for usability here having one tests makes sense since it was faster for me when running locally haha.